### DISCUSSION TITLE
Hosting with docker?

### DISCUSSION THREAD

**Discussion by landonhalley** [jagrosh/MusicBot#766] [Questions]
Is there any easy way I can host this in docker? I am having a hard time figuring out how to do so

**Comment by MichailiK** [comment]
[There](https://github.com/jagrosh/MusicBot/pull/771) [are](https://github.com/jagrosh/MusicBot/pull/736) a [few](https://github.com/jagrosh/MusicBot/pull/650) [PR's](https://github.com/jagrosh/MusicBot/pull/536) for a Dockerfile, none of which have been accepted yet. If you want, you can try those yourself & see if they work. Do note that there's no support provided for any of these Docker implementations.
